### ISSUE TITLE
Cloning an ImageStyle doesn't clone Icon displacement

### ISSUE THREAD

**Issue by mediafreakch** [openlayers/openlayers#13802]
**Describe the bug**

When cloning a Style containing an image, the `displacement` property of the Icon isn't cloned with it:

```js
const style = new Style({
   image: new Icon({
      src: 'path/to/svg',
      scale: 1.2,
      displacement: [10, 50],
   }),
   zIndex: 11,
})
```

Seems to be [this line](https://github.com/openlayers/openlayers/blob/v6.14.1/src/ol/style/Icon.js#L220)

**To Reproduce**
- 

**Expected behavior**

The displacement should be cloned from the original style.

**Comment by mike-000** [comment]
Yes, it was probably overlooked because `getAnchor()` returns the anchor converted to pixels and adjusted for displacemnent.  But when cloning the original anchor is used, so displacement is also needed as in the parent Image class: `displacement: this.getDisplacement().slice(),`.
